src/System.Private.ServiceModel/tools/scripts/PingWCFService.ps1

```diff
@@ -0,0 +1,19 @@
+﻿param (
+    [Parameter(Mandatory=$true)]
+    [string] $Url
+)
+$request = New-Object System.Net.WebCLient
+For ($i=0; $i -le 10; $i++)
+{
+    try
+    {
+        $request.DownloadFile($Url, "temp.crl")
```

Comment:
Agree to consider for the future -- incidentally, having a ping address I could hit using 'curl' from Linux or Fiddler was instrumental in testing whether the services were available.  I'll open an issue for it